src/core/Panel.ts

```diff
@@ -202,6 +150,21 @@ class Panel {
     return this;
   }
 
+  public increaseOffset(val: number): this {
+    this._offset += val;
+    return this;
+  }
+
+  public decreaseOffset(val: number): this {
+    this._offset -= val;
+    return this;
+  }
+
+  public resetOffset(): this {
+    this._offset = 0;
+    return this;
+  }
```

Comment:
How about making it with a common logic like below?

```ts
public increaseOffset(val: number): this {
    return _setOffset(val);;
}

public decreaseOffset(val: number): this {
    return _setOffset(-val);
}

public resetOffset(): this {
    return _setOffset(0);
}

private _setOffset(val:number): this {
    this._offset = val;
    return this;
}
```

Reply:
Actually, this offset thing is a temporary implementation of the camera offset, and I'm wondering how to deal with it more properly.
As we can't modify the actual animation offset as we can't change Axes's animation value, I adopted this offset but I don't believe this is the perfect way for doing that.
So anyway, I just want to tell you that this offset can be removed somehow future.